File: LeetCode-Programs/Length of Last word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
         int length=0;
        for (int i=s.size()-1;i>= 0;i--) 
        {
            if (s[i]!=' ') 
                length=length+1;
            if (length>0 && s[i]==' ')
                break;
        }
        return length;
        }
};

File: LeetCode-Programs/Length of Last word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        length=0
        for i in range(len(s)-1, -1, -1):
            if s[i]!=' ':
                length=length+1
            if length>0 and s[i]==' ':
                break
        return length
